core/src/main/java/org/elasticsearch/plugins/PluginCli.java

```diff
@@ -40,7 +39,20 @@ public PluginCli(Environment env) {
     }
 
     public static void main(String[] args) throws Exception {
-        BasicConfigurator.configure(new NullAppender());
+        // initialize default for es.logger.level because we will not read the logging.yml
```

Comment:
We did have a test for this - to make sure we didn't start cramming logs into eshome/logs when you run the plugin installer from the deb/rpm. Since it has to be run as root it can really be a bull in a china shop. Anyway, I'd love for us to be able to get debug logs from this process if we want them. Since we don't forward these logs through the Terminal object that we use to test the installer's output this particular change could pollute that output. The PR has tests for that in the bats tests but the feedback cycle isn't good for those - they just take a long time. Even when we get them running in CI they will still take time. And I can image failing the build with strange, action at a distance style logging failures. Which would suck.

Anyway, I see a couple of options:
1. Move this into another PR and do nothing about it here.
2. Keep this as is.
3. Keep this as is but change the default level to `WARN` or `ERROR` so it is less likely to spit out extra stuff.
4. Try to figure out where logging.yaml is from here and load it. I'm not even sure this is right because we're not actually the elasticsearch server in this context - we're the plugin installer. So the logging config may be bogus.

Personally I prefer option 3. I *want* the debugging output to go to the terminal for something like this.